dataland-frontend/tests/e2e/utils/LksgUpload.ts

```diff
@@ -264,6 +266,22 @@ function checkIfUploadFieldDependenciesAreRespected(): void {
   cy.get("input[name=capacity]").should("not.exist");
 }
 
+/**
+ *  Verify that selected documents are referenced in the actual dataset
+ */
+function checkIfUploadedFilesAreReferencedInTheDataset(): void {
+  cy.intercept("POST", "**/api/data/lksg").as("postLksgData");
+  submitButton.clickButton();
+  cy.wait("@postLksgData").then((interception) => {
+    const postedObject = interception.request.body as CompanyAssociatedDataLksgData;
+    const postedLksgDataset = postedObject;
+    const referencedReports =
```

Comment:
This is not a whole report.  It is just the hash of a report.

Reply:
Changed to referencedReportHash